src/components/common/balance/BalanceDisplayUSD.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { getRoundedBalanceInFiat } from './helpers';
+
+const BalanceDisplayUSD = ({
+    amount,
+    showAlmostEqualSignUSD = true,
+    showSignUSD = false,
+    showSymbolUSD = true,
+    nearTokenFiatValueUSD
+}) => {
+
+    const roundedBalanceInUSD = getRoundedBalanceInFiat(amount, nearTokenFiatValueUSD);
+    const amountPrefix = roundedBalanceInUSD !== '< 0.01' ? '≈ ' : '';
```

Comment:
Feels weird that we have a prop named `showAlmostEqualSignUSD`, but then we have `amountPrefix` be replaced with an empty string even if `showAlmostEqualSignUSD` is true. 🤷

Reply:
Good catch:) Fixing.